subprojects/core/src/main/java/org/gradle/internal/service/scopes/TaskExecutionServices.java

```diff
@@ -139,17 +143,20 @@ TaskExecuter createTaskExecuter(TaskArtifactStateRepository repository, TaskOutp
         );
     }
 
-    private static TaskExecuter createSkipCachedExecuterIfNecessary(StartParameter startParameter, BuildCacheConfigurationInternal buildCacheConfiguration, TaskOutputPacker packer, TaskOutputsGenerationListener taskOutputsGenerationListener, TaskOutputOriginFactory taskOutputOriginFactory, TaskExecuter delegate) {
+    private static TaskExecuter createSkipCachedExecuterIfNecessary(StartParameter startParameter, BuildCacheConfigurationInternal buildCacheConfiguration, TaskOutputPacker packer, TaskOutputsGenerationListener taskOutputsGenerationListener, TaskOutputOriginFactory taskOutputOriginFactory, ListenerManager listenerManager, TaskExecuter delegate) {
         if (startParameter.isTaskOutputCacheEnabled()) {
```

Comment:
We should do this regardless of whether the cache is being used. This information is also very valuable when the cache is not being used.

We should also sync this up with the 'out-of-date' messages, so the same information is presented there.